modules/admission-controller-about.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-controller-about_{context}"]
+= About admission controllers
+
+Admission controllers are used to help regulate how {product-title} {product-version} functions.  Admission control plug-ins intercept requests to the master API to validate resource requests and ensure policies are adhered to. For example, admission control plug-ins are commonly used to enforce security policy, resource limitations or configuration requirements.
+
+Admission control plug-ins run in an admission chain, starting with mutating controllers that can modify resources, followed by validating controllers. Some controllers can both modify and validate. If any controller in the sequence rejects a request, the whole chain is aborted and an error is returned. Mutating controllers run prior to validating controllers so that modifications that they trigger can also be validated.
+
+{product-title} has a default set of plug-ins enabled for each resource type. These are required for proper functioning of master nodes. In addition, administrators can define custom admission webhooks to call webhook servers that enforce custom policies outside of these defaults.
```

Comment:
> These are required for proper functioning of master nodes.

Sounds strange. Why is this formulated so node centric?

Reply:
This has now been reworded to be cluster centric not node centric.